vboxtouch/vboxtouch.h

```diff
@@ -61,11 +61,19 @@ private slots:
     QRect m_screenGeometry; // x,y - ui offset relative to framebuffer; size - framebuffer size
     ZoomIndicator *m_indicator;
 
+    //! An eventfd object for signal handling
+    static int s_quitSignalFd;
+    //! Socket notifier for signal handling
```

Comment:
I'd consider omitting the comments as the surrounding code is quite _reserved_ at commenting and does not use qdoc comments at all :)

Reply:
Done.